lesson-02/hw2/example-03.js

```diff
@@ -0,0 +1,21 @@
+const findLongestWord = function(string) {
+  let wordLength = 0;
+  let longestWord;
+  const STRINGARR = string.split(' ');
+  for (let i = 0; i < STRINGARR.length; i += 1) {
+    if (STRINGARR[i].length > wordLength) {
```

Comment:
Теж можна скороти кількість змінних :) 
```suggestion
    if (STRINGARR[i].length > longestWord.length) {
      longestWord = STRINGARR[i];
    }
```